src/utils/classToAtom.js

```diff
@@ -0,0 +1,31 @@
+const atomRenderer = (component) => ({env, options, payload}) => {
+  const {onTeardown} = env;
+
+  const element = React.createElement(component, {
+    ...env,
+    ...options,
```

Comment:
I'm actually not sure! The options description says `options [object] - Rendering options that were passed to the renderer (as atomOptions) when it was instantiated`. I'm not doing anything with `atomOptions` at the moment, so I'm personally tempted to just ignore the options argument altogether. But I feel like it might be helpful to pass through everything just in case? What do you think?